ipfix-flow-exporter/bpf_ipfix_egress.bpf.c

```diff
@@ -3,7 +3,7 @@
 
 #define KBUILD_MODNAME "foo"
 
-#include "../headers/vmlinux.h"
+#include "./vmlinux.h"
```

Comment:
IMHO 
```suggestion
#include "vmlinux.h"
```
add flag "-I." in compiler